string drone_name
float32 prev_x
float32 prev_y
float32 prev_z
float32 prev_yaw
float32 prev_pitch
float32 prev_roll
---
float32 x
float32 y
float32 z
float32 yaw
float32 pitch
float32 roll
bool notvalidflag
